### PR TITLE
Bug 1832832: Fix disk extension

### PR DESCRIPTION
**Pull request by rgolangh**
Disk extension was failing because of wrong API usage.

### CODE REVIEW

**Comment by rgolangh** [comment]
@bennyz PTAL

File: pkg/cloud/ovirt/clients/machineservice.go [hunk 1]
```diff
@@ -180,20 +180,27 @@ func (is *InstanceService) handleDiskExtension(vmService *ovirtsdk.VmService, cr
...
-			newDiskSize)
 
+	// get the disk
+	getDisk, err := vmService.Connection().SystemService().DisksService().DiskService(bootableDiskAttachment.MustId()).Get().Send()
+	if err != nil {
+		return err
+	}
+
+	size := getDisk.MustDisk().MustProvisionedSize()
+	if newDiskSize > size {
```
Comment (Gal-Zaidman) [inline pkg/cloud/ovirt/clients/machineservice.go:191]: I think we should add a elif newDiskSize < size --> log that shrinking is not supported

Comment (rgolangh) [reply pkg/cloud/ovirt/clients/machineservice.go:195]: done

**Pull request merged**